PrintingPodRecharge/Patches/LocalizationPatch.cs

```diff
@@ -10,7 +10,7 @@ public class Localization_Initialize_Patch
         {
             public static void Postfix()
             {
-                Loc.Translate(typeof(STRINGS), true);
+                Translations.RegisterForTranslation(typeof(STRINGS), true);
             }
```

Comment:
I have absolutely no idea what I am doing here. It just complained that `Loc` does not exist (probably ONI changed the `Localization` class in an update recently), so I applied the same change you made to MoreMarbleSculptures [here](https://github.com/aki-art/ONI-Mods/commit/19a245cdbe2ce5367923a03e2a52a00d7698086f#diff-b6c3810ccf54168f7c52ca907348d8c8b5c5986b4632b531ae7cbeb1678f0a43)

Reply:
Also outdated Futility